chanzuckerberg/idseq-web
# Update css-loader to version ^3.4.2 (IDSEQ-2216)

# Description

This PR updates css-loader to the newest version. Breaking changes in config were introduced, so some webpack configuration has been updated for the latest version.

# Notes

The `minimize` option for css-loader was deprecated; this has been replaced by adding `postcss-loader` to the webpack build, with the `cssnano` plugin option.

# Tests

* Run `npm start` to ensure webpack builds correctly
* Navigate around all of idseq to ensure CSS is working properly
